Neos.Fusion/Tests/Unit/Core/ParserTest.php

```diff
@@ -63,7 +64,7 @@ public function parserCorrectlyParsesFixture01()
             ]
         ];
 
-        $actualParseTree = $this->parser->parse($sourceCode);
+        $actualParseTree = $this->parser->parseFrom(\Neos\Fusion\Core\FusionSourceCode::fromString($sourceCode));
```

Comment:
I think at least some of the old parse* tests should stay there for now so we know that those do still work as before until 9.0 ... otherwise we may break stuff for packages like monocle without noticing

Reply:
yes youre right ;) ill add an extra test file which can later be removed ;)